sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/ApplicationInsightsRestClient.cs

```diff
@@ -42,7 +42,7 @@ internal async Task<HttpMessage> InternalTrackAsync(IEnumerable<TelemetryItem> b
             catch (Exception ex)
             {
                 AzureMonitorExporterEventSource.Log.FailedToTransmit(ex);
-                if (ex.InnerException?.Source != "System.Net.Http")
+                if (ex.InnerException?.Source != "System.Net.Http" && ex.InnerException?.Source != "System")
```

Comment:
Why are we adding an additional check here?

Reply:
for `net462` target the source value is different.